packages/cubejs-playground/src/components/Header.tsx

```diff
@@ -128,6 +128,15 @@ const Header = ({ selectedKeys }) => {
             <SlackOutlined />
             Slack
           </StyledMenuButton>
+          <StyledMenuButton
+            noMargin
+            key="discourse"
+            href="https://forum.cube.dev/"
+            target="_blank"
+          >
+            <component={DiscourseIcon} />
```

Comment:
Hmm... @vasilev-alex this is causing an error and I also tried `< DiscourseIcon />`. 

How should I correct this, assuming everything in `DiscourseIcon.tsx` is correct?

Reply:
This should do it:

```suggestion
            <DiscourseIcon />
```